pkg/test/auth/tokenmanager_test.go

```diff
@@ -173,3 +173,23 @@ func TestTokenManagerKeyService(t *testing.T) {
 		require.Equal(t, key1.N, rsaKey1.N)
 	})
 }
+
+func TestTokenManagerE2ETestKeys(t *testing.T) {
+
+	t.Run("test get e2e token", func(t *testing.T) {
+		identity := NewIdentity()
+		emailClaim := WithEmailClaim(uuid.NewV4().String() + "@email.tld")
+		token, err := GenerateSignedE2ETestToken(*identity, emailClaim)
+		require.NoError(t, err)
+		require.NotNil(t, token)
+	})
+
+	t.Run("test rsa public key", func(t *testing.T) {
+		publicKey := GetE2ETestPublicKey()[0]
```

Comment:
As I mentioned in the previous review I would merged these two tests:
```
	identity := NewIdentity()
	emailClaim := WithEmailClaim(uuid.NewV4().String() + "@email.tld")
	token, err := GenerateSignedE2ETestToken(*identity, emailClaim)
	require.NoError(t, err)
	require.NotNil(t, token)
	validateToken(t, token)
...

func validateToken(t *testing.T, token string) {
	publicKeys := GetE2ETestPublicKey()
	require.Len(t, publikKeys, 1) 
	publicKey := publicKeys[0]
	parsedToken, err := jwt.Parse(token, &TokenClaims{}, func(token *jwt.Token) (interface{}, error) {
		kid := token.Header["kid"]
		require.NotNil(t, kid)
		kidStr, ok := kid.(string)
		require.True(t, ok)
		assert.Equal(t, publicKey.KeyID, kidStr)
		return publicKey.Key, nil
	})
	require.NoError(t, err)
	assert.True(t, parsedToken.Valid())
}
```

Basically we generate a token and validate it using the public key. Then we can make sure that both functions use the same private key behind the scenes.

Reply:
done here: 37f6a8660df0f08627cfbf6239c0d699957922e2